assets/registry/store/@linclark/pkg/package.json

```diff
@@ -1 +1 @@
+{"_id":"@linclark/pkg","_rev":"3-6b1588cc0c82adb7a1dd3edaa5b24d92","name":"@linclark/pkg","description":"An empty package for the how-to-npm workshopper to use.","dist-tags":{"latest":"1.0.2"},"versions":{"1.0.0":{"name":"@linclark/pkg","version":"1.0.0","description":"An empty package for the how-to-npm workshopper to use.","author":{"name":"Lin Clark"},"license":"ISC","_id":"@linclark/pkg@1.0.0","scripts":{},"_shasum":"c741209d9f8e959168f4a698c6d569212054db7c","_from":".","_npmVersion":"2.9.1","_nodeVersion":"0.10.35","_npmUser":{"name":"linclark","email":"lin.w.clark@gmail.com"},"dist":{"shasum":"c741209d9f8e959168f4a698c6d569212054db7c","tarball":"http://registry.npmjs.org/@linclark/pkg/-/pkg-1.0.0.tgz"},"maintainers":[{"name":"linclark","email":"lin.w.clark@gmail.com"}],"directories":{}},"1.0.1":{"name":"@linclark/pkg","version":"1.0.1","description":"An empty package for the how-to-npm workshopper to use.","author":{"name":"Lin Clark"},"license":"ISC","_id":"@linclark/pkg@1.0.1","scripts":{},"_shasum":"2f10c9ae04e61a7efa9fb43097352808321705a0","_from":".","_npmVersion":"2.9.1","_nodeVersion":"0.10.35","_npmUser":{"name":"linclark","email":"lin.w.clark@gmail.com"},"dist":{"shasum":"2f10c9ae04e61a7efa9fb43097352808321705a0","tarball":"http://registry.npmjs.org/@linclark/pkg/-/pkg-1.0.1.tgz"},"maintainers":[{"name":"linclark","email":"lin.w.clark@gmail.com"}],"directories":{}},"1.0.2":{"name":"@linclark/pkg","version":"1.0.2","description":"An empty package for the how-to-npm workshopper to use.","main":"index.js","author":{"name":"Lin Clark"},"license":"ISC","_id":"@linclark/pkg@1.0.2","scripts":{},"_shasum":"d031778d174e04874a2c8eaba5928898a1d50292","_from":".","_npmVersion":"2.9.1","_nodeVersion":"0.10.35","_npmUser":{"name":"linclark","email":"lin.w.clark@gmail.com"},"dist":{"shasum":"d031778d174e04874a2c8eaba5928898a1d50292","tarball":"http://registry.npmjs.org/@linclark/pkg/-/pkg-1.0.2.tgz"},"maintainers":[{"name":"linclark","email":"lin.w.clark@gmail.com"}],"directories":{}}},"readme":"ERROR: No README data found!","maintainers":[{"name":"linclark","email":"lin.w.clark@gmail.com"}],"time":{"modified":"2015-05-21T20:16:45.507Z","created":"2015-05-21T16:34:50.033Z","1.0.0":"2015-05-21T16:34:50.033Z","1.0.1":"2015-05-21T16:36:33.580Z","1.0.2":"2015-05-21T20:16:45.507Z"},"author":{"name":"Lin Clark"},"license":"ISC","readmeFilename":"","_attachments":{}}
```

Comment:
Not sure why this file is modified?

Reply:
The files were just swapped, I fixed that.